#!/usr/bin/env python
PACKAGE = "robotics_odometry_project"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

i_m_enum = gen.enum([ gen.const("euler", int_t, 0, "A very basic integration method"),
                       gen.const("rk", int_t, 1, "A bit more complex integration method")],
                     "An enum to set the integration method")

gen.add("int_meth", int_t, 0, "The method used to compute the odometry", 0, 0, 1, edit_method=i_m_enum)

exit(gen.generate(PACKAGE, "robot_main_node", "parameters"))
